packages/orchestrator-ui-components/src/components/WfoForms/formFields/ListAddField.tsx

```diff
@@ -53,9 +53,9 @@ function ListAdd({
         !disabled &&
         !(
             parent.maxCount! <=
-            Math.max(initialCount ?? 0, parent.value!.length)
+            Math.max(initialCount ?? 0, parent?.value!.length)
         );
-    const count = 1 + Math.max((initialCount ?? 0) - parent.value!.length, 0);
+    const count = 1 + Math.max((initialCount ?? 0) - parent?.value!.length, 0);
```

Comment:
Could maybe be combined with a single assignment to a var and resused in 56?

Reply:
Doesn't do the same thing... I think